helix-core/src/main/java/org/apache/helix/HelixProperty.java

```diff
@@ -156,25 +156,50 @@ public String toString() {
    * @param id
    */
   public HelixProperty(String id) {
-    this(new ZNRecord(id), id);
+    this(new ZNRecord(id), id, false);
```

Comment:
@junkaixue default value is false here because there is no need to make a copy of an `ZNRecord` the caller hasn't passed in.